serving-sdk/src/test/java/software/amazon/ai/mms/servingsdk/impl/ModelImpl.java

```diff
@@ -0,0 +1,38 @@
+/*
+ *  Copyright (c) 2019 Amazon.com, Inc. or its affiliates.
+ *  All Rights Reserved. Licensed under the Apache License, Version 2.0 (the "License").
+ *  You may not use this file except in compliance with the License. A copy of the License is located at
+ *  http://aws.amazon.com/apache2.0/ or in the "license" file accompanying this file. This file is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License forthe specific language governing permissions and limitations under the License.
+ */
+
+package software.amazon.ai.mms.servingsdk.impl;
+
+import software.amazon.ai.mms.servingsdk.Model;
+import software.amazon.ai.mms.servingsdk.Worker;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ModelImpl implements Model {
```

Comment:
Better call it MockModel if you prefer write your own Mock class.

Reply:
Makes sense.

Reply:
Just documenting our discussion. The initial thought here was to make sure that we have both positive and negative cases. I modified these tests to have Mock and Reflections in the tests instead now.